src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,57 @@
+// import React, { useState } from "react";
+// import { Drawer } from "antd";
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "../../store/store";
+import { changeToCartPage, changeToBookPage } from "../../domain/app/appSlice";
+import { useNavigate } from "react-router-dom";
+import "./style.css";
+
+const Navbar = () => {
+  // const [open, setOpen] = useState(false);
+  const cartdata = useSelector((state: RootState) => state.cart.cart);
```

Comment:
Do not write the selectors here, instead move it to its respective slice.
Inside cartslice.ts, create a cartSelector just like in booksSlice.

Reply:
sure